src/components/gallery/HdGalleryMedia.vue

```diff
@@ -62,23 +65,18 @@ export default {
   watch: {
     item() {
       this.$nextTick(() => {
-        this.hideThumbnailOnBgLoad();
+        this.hideThumbnail();
```

Comment:
```suggestion
```
This one can be removed because we are doing the opposite in the following line.